src/AzSK.ADO/Framework/Core/SVT/ADO/ADO.ServiceConnection.ps1

```diff
@@ -1059,4 +1059,125 @@ class ServiceConnection: ADOSVTBase
 
         return $controlResult;
     }
+
+    hidden [ControlResult] CheckTemplateBranchForSvcConn ([ControlResult] $controlResult) {
+        try{            
+            $url = "https://dev.azure.com/{0}/{1}/_apis/pipelines/checks/queryconfigurations?`$expand=settings&api-version=6.1-preview.1" -f $this.OrganizationContext.OrganizationName, $this.ResourceContext.ResourceGroupName;
+            #using ps invoke web request instead of helper method, as post body (json array) not supported in helper method
+            $rmContext = [ContextHelper]::GetCurrentContext();
+            $user = "";
+            $base64AuthInfo = [Convert]::ToBase64String([Text.Encoding]::ASCII.GetBytes(("{0}:{1}" -f $user,$rmContext.AccessToken)))  
+            $body = "[{'name':  '$($this.ResourceContext.ResourceDetails.Name)','id':  '$($this.ResourceContext.ResourceDetails.Id)','type':  'endpoint'}]"
+            $response = @(Invoke-RestMethod -Uri $url -Method Post -ContentType "application/json" -Headers @{Authorization=("Basic {0}" -f $base64AuthInfo)} -Body $body)
+            if([Helpers]::CheckMember($response, "count") -and $response[0].count -eq 0){
```

Comment:
not need for this [Helpers]::CheckMember($response, "count") , count always will be there